src/ui-kit/form-controls/date/date.component.ts

```diff
@@ -490,19 +499,19 @@ export class SamDateComponent
   }
 
   triggerTouch() {
-    this.onTouched();
+    // this.onTouched();
   }
 
   triggerMonthTouch(event) {
-    if (event.target.value.substring(0, 1) === '0') {
-      this.month.nativeElement.value = event.target.value.substring(1);
-    }
+    // if (event.target.value.substring(0, 1) === '0') {
```

Comment:
@yerramshilpa Same here -- if you don't need this code, delete it rather than commenting it out.

Reply:
Or don't call `triggerMonthTouch`, but since that is a public property, we might be introducing breaking changes to a very highly used component.

Reply:
forgot to uncomment